client/src/main/java/org/apache/rocketmq/client/producer/LocalTransactionExecutor.java

```diff
@@ -18,6 +18,6 @@
 
 import org.apache.rocketmq.common.message.Message;
 
-public interface LocalTransactionExecuter {
-    public LocalTransactionState executeLocalTransactionBranch(final Message msg, final Object arg);
+public interface LocalTransactionExecutor {
```

Comment:
So in the next version, we must announce that they may face the compatible problem when they use XXXX in the older version.

Before they update, they must know about this and know how to fix them , and then upgrade.